svg/attributes/core.json

```diff
@@ -262,10 +262,10 @@
                 "version_added": null
               },
               "firefox": {
-                "version_added": null
+                "version_added": "3"
               },
               "firefox_android": {
-                "version_added": null
+                "version_added": true
```

Comment:
This should be `"4"`, as Firefox for Android began with version 4, and included almost, if not all features from Firefox 1‑4.